src/main/java/edu/greenblitz/tobyDetermined/commands/telescopicArm/claw/GripFromClaw.java

```diff
@@ -0,0 +1,23 @@
+package edu.greenblitz.tobyDetermined.commands.telescopicArm.claw;
+
+import edu.wpi.first.wpilibj.Timer;
+
+public class GripFromClaw extends ClawCommand {
+    private static final double gripTime = 2; //seconds
```

Comment:
both this and eject from claw should allow for both a constant number and a parameter using overloading